lib/measured/conversion_builder.rb

```diff
@@ -0,0 +1,35 @@
+class Measured::ConversionBuilder
+  def initialize
+    @base_unit = nil
+    @units = []
+    @case_sensitive = false
+  end
+
+  def base(unit_name, aliases: [])
+    raise Measured::UnitError, "Setting #{unit_name} as base unit, but already defined as #{@base_unit}." if @base_unit
+    @base_unit = Measured::Unit.new(unit_name, aliases: aliases)
+  end
+
+  def unit(unit_name, aliases: [], value:)
+    @units << Measured::Unit.new(unit_name, aliases: aliases, value: value)
+  end
+
+  def case_sensitive(is_case_sensitive)
+    @case_sensitive = is_case_sensitive
```

Comment:
I thought we should only avoid `is_` for function names, since we have question mark suffixes, but local variables would be fine? Wouldn't naming something `adjective` over `is_adjective` (if no better option exists) go against the whole "code should document itself"? For example, `blue` and `is_blue` would be two very different variables.

^ Just for curiosity because I'm not sure I agree with the "always avoid". I'm good with this specific instance :)